content/blog/extracing-rpm-build-depends.md

```diff
@@ -0,0 +1,194 @@
++++
+title = 'Extracing RPM Build Dependencies From RPM Spec Files'
+date = 2019-01-08T19:47:19-08:00
+draft = true
+tags = ["tags"]
+description = "Desc"
+
+# For twitter cards, see https://github.com/mtn/cocoa-eh-hugo-theme/wiki/Twitter-cards
+meta_img = "/images/image.jpg"
+
+# For hacker news and lobsters builtin links, see github.com/mtn/cocoa-eh-hugo-theme/wiki/Social-Links
+hacker_news_id = ""
+lobsters_id = ""
++++
+
+# Specifying build dependencies for RPM packages
+
+Build dependencies in an RPM spec are denoted by `BuildRequires: ${DEPENDENCY}`.
+
+You'll typically find a random spec file with dependencies like this:
+```config
+Name: foo
+
+BuildRequires: make
+BuildRequires: bar
+
+# We're running something on a SuSE distribution
+%if 0%{?suse_version}
+BuildRequires: suse-specific-stuff
+%else
+BuildRequires: not-on-suse
+%endif
+```
+
+# The normal way to extract dependencies
+
+So for every tutorial that you see to build an RPM package one tool stands
+above all the rest when it comes to extract / install build dependencies:
+
+```
+$ yum-builddep
+```
+
+## But how do you run it?
+
+```
+$ yum-builddep -y ${SPEC_FILE}
+```
+
+## Okay that was too easy, why even write a blog post about this?
+
+Well, my traveler, for all the good that `yum-builddep` does, it has the
+most common flaw of them all: **it's not available everywhere**.
+
+For a tool as good as it is, it's reliance on underlying package managers
```

Comment:
I think it's should be its

Reply:
Never mind I think it reads correctly as is.

Reply:
switched `it's` to `the`